components/prism-wolfram.js

```diff
@@ -0,0 +1,29 @@
|$HTTPRequest|$IgnoreEOF|$ImageFormattingWidth|$ImageResolution|$ImagingDevice|$ImagingDevices|$ImportFormats|$IncomingMailSettings|$InitialDirectory|$Initialization|$InitializationContexts|$Input|$InputFileName|$InputStreamMethods|$Inspector|$InstallationDate|$InstallationDirectory|$InterfaceEnvironment|$InterpreterTypes|$IterationLimit|$KernelCount|$KernelID|$Language|$LaunchDirectory|$LibraryPath|$LicenseExpirationDate|$LicenseID|$LicenseProcesses|$LicenseServer|$LicenseSubprocesses|$LicenseType|$Line|$Linked|$LinkSupported|$LoadedFiles|$LocalBase|$LocalSymbolBase|$MachineAddresses|$MachineDomain|$MachineDomains|$MachineEpsilon|$MachineID|$MachineName|$MachinePrecision|$MachineType|$MaxExtraPrecision|$MaxLicenseProcesses|$MaxLicenseSubprocesses|$MaxMachineNumber|$MaxNumber|$MaxPiecewiseCases|$MaxPrecision|$MaxRootDegree|$MessageGroups|$MessageList|$MessagePrePrint|$Messages|$MinMachineNumber|$MinNumber|$MinorReleaseNumber|$MinPrecision|$MobilePhone|$ModuleNumber|$NetworkConnected|$NetworkInterfaces|$NetworkLicense|$NewMessage|$NewOptimizations|$NewSymbol|$NotebookInlineStorageLimit|$Notebooks|$NoValue|$NumberMarks|$NumberOfExtraPages|$Off|$OperatingSystem|$Output|$OutputForms|$OutputSizeLimit|$OutputStreamMethods|$Packages|$ParentLink|$ParentProcessID|$PasswordFile|$PatchLevelID|$Path|$PathnameSeparator|$PerformanceGoal|$Permissions|$PermissionsGroupBase|$PersistenceBase|$PersistencePath|$PipeSupported|$PlotTheme|$Post|$Pre|$PreferencesDirectory|$PreInitialization|$PrePrint|$PreRead|$PrintForms|$PrintLiteral|$Printout3DPreviewer|$ProcessID|$ProcessorCount|$ProcessorType|$ProductInformation|$ProgramName|$ProgressReporting|$PublisherID|$RandomGeneratorState|$RandomState|$RecursionLimit|$RegisteredDeviceClasses|$RegisteredUserName|$ReleaseNumber|$RequesterAddress|$RequesterWolframID|$RequesterWolframUUID|$ResourceSystemBase|$ResourceSystemPath|$RootDirectory|$ScheduledTask|$ScriptCommandLine|$ScriptInputString|$SearchLanguage|$SecuredAuthenticationKeyTokens|$ServiceCreditsAvailable|$Services|$SessionID|$SetParentLink|$SharedFunctions|$SharedVariables|$SoundDisplay|$SoundDisplayFunction|$SourceLink|$SSHAuthentication|$SubtitleDecoders|$SubtitleEncoders|$SummaryBoxDataSizeLimit|$SuppressInputFormHeads|$SynchronousEvaluation|$SyntaxHandler|$System|$SystemCharacterEncoding|$SystemCredentialStore|$SystemID|$SystemMemory|$SystemShell|$SystemTimeZone|$SystemWordLength|$TargetSystems|$TemplatePath|$TemporaryDirectory|$TemporaryPrefix|$TestFileName|$TextStyle|$TimedOut|$TimeUnit|$TimeZone|$TimeZoneEntity|$TopDirectory|$TraceOff|$TraceOn|$TracePattern|$TracePostAction|$TracePreAction|$UnitSystem|$Urgent|$UserAddOnsDirectory|$UserAgentLanguages|$UserAgentMachine|$UserAgentName|$UserAgentOperatingSystem|$UserAgentString|$UserAgentVersion|$UserBaseDirectory|$UserBasePacletsDirectory|$UserDocumentsDirectory|$Username|$UserName|$UserURLBase|$Version|$VersionNumber|$VideoDecoders|$VideoEncoders|$VoiceStyles|$WolframDocumentsDirectory|$WolframID|$WolframUUID)\b/,
```

Comment:
Can't we remove all `$XXX` variables from this now that we have `global-variable`?

```suggestion
	'keyword': /\b(?:Abs|AbsArg|Accuracy|Block|Do|For|Function|If|Manipulate|Module|Nest|NestList|None|Return|Switch|Table|Which|While)\b/,
```

Reply:
Ah yes, we don't need those now.